CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing to Gymkhana
+
+We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting an issue
+- Submitting a fix
+- Proposing new features
+
+## Standard Commit Messages
+
+This project is using the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0-beta.2/) standard. Please follow these steps to ensure your
+commit messages are standardized:
+
+- Commit messages should have this format:
+  `<type>[optional scope]: <description>`
+- Type should be `fix`(if fixed some bug), `feat`(if added new feature), `docs`(changes in documentation) or `test`(added new tests)
```

Comment:
Let's add `chore` as well for dependency/configuration change. Later if need arises we can update this list to have more.